docker/superset/versions.yaml

```diff
@@ -16,19 +16,19 @@
 cloudbuild:
   enable_parallel: false
 versions:
-- dir: 2/debian11/2.1
```

Comment:
@AdrianManiek nice you added the version 3, but instead of removing the version 2, it would be better to firstly upgrade the version 2.1.0 to 2.1.1 and also add the version 3. Then we would be able to upgrade the K8s and later on - after publishing the container 3.x then handle the K8s upgrade to version 3